packages/raccoon-city-ui/src/components/Main/Contacts/Contact/ContactTrades.tsx

```diff
@@ -0,0 +1,249 @@
+import React from 'react';
+import {useQuery} from '@apollo/react-hooks';
+import {Accordion, AccordionSummary, AccordionDetails} from '@material-ui/core';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import Typography from '@material-ui/core/Typography';
+import {GET_TRADE_DROPDOWNS} from '../../../../graphql/queries/tradeQuery';
+import {format} from 'date-fns';
+import {getConstant, getClientInterests} from './ContactTradesUtils';
+import {RootContainer, TradeTitleContainer, Bold} from './ContactForm.styled';
+
+interface Props {
+    trades: any;
+}
+
+const ContactTrades: React.FC<Props> = ({trades}) => {
+    const {data: dropdowns, loading: dropdownsLoading} = useQuery(GET_TRADE_DROPDOWNS);
+
+    if (!trades || dropdownsLoading) {
+        return <div>Loading</div>;
+    }
+
+    return (
+        <RootContainer>
+            {trades.map((item) => (
+                <Accordion key={item.id} TransitionProps={{unmountOnExit: true}}>
+                    <AccordionSummary aria-controls="panel1a-content" id={item.id}>
+                        <TradeTitleContainer>
+                            <p>
+                                <Bold>№ :</Bold>&nbsp;{item.tradeNumber}
+                                <Bold>Статус :</Bold>&nbsp;{getConstant(dropdowns, item.leadStatus, 'leadStatuses')}
+                            </p>
+                            <p>
+                                <Bold>ЖК :</Bold>
+                                {item.flat.apartmentComplex}
+                                <Bold>Дом :</Bold>
+                                {item.flat.house}
+                                <Bold>Квартира :</Bold>
+                                {item.flat.flatNumber}
+                            </p>
```

Comment:
Could you please in subsequent PRs attach screenshots?

Reply:
yep